examples/bert_inc_ptq_cpu/README.md

```diff
@@ -14,7 +14,23 @@ python -m pip install -r requirements.txt
 ```
 
 ## Run sample
-### Run with both Intel® Neural Compressor static and dynamic quantization
+### Run Intel® Neural Compressor quantization with or without accuracy aware tuning
+
+Accuracy aware tuning is one of unique features provided by Intel® Neural Compressor quantization. This feature can be used to solve accuracy loss pain points brought by applying low precision quantization and other lossy optimization methods. Intel® Neural Compressor also support to quantize all quantizable ops without accuracy tuning, user can decide whether to tune the model accuracy or not. Please check the [doc](https://github.com/intel/neural-compressor/blob/master/docs/source/quantization.md) for more details.
+
+User can decide to tune the model accuracy by setting 'evaluate_func'. If not set, accuracy of the model will not be tuned.
+
+```json
+"quantization": {
+    "type": "IncQuantization",
+    "config": {
+        "user_script": "user_script.py",
+        "evaluate_func": "eval_accuracy",

```

Comment:
Does that say if user choose the build-in accuracy metrics in the part of `olive.evaluator` , they still need to provide `evaluate_func` here? 
Also the the `accuracy_criterion` looks a bit duplicated with olive's metric goal. 

If we can leverage the olive.evaluator metrics here Instead of use `evaluate_func`?